sdk/servicebus/azure-servicebus/samples/async_samples/authenticate_using_azure_sas_credential_async.py

```diff
@@ -17,7 +17,7 @@
 try:
     from urllib.parse import quote as url_parse_quote
 except ImportError:
-    from urllib import pathname2url as url_parse_quote
+    from urllib.request import pathname2url as url_parse_quote # type: ignore[no-redef]
```

Comment:
the try/except can be removed and simply have the import on line 18. This was needed for py2 compatibility 🙂